src/neural/BLAS/transforms.h

```diff
@@ -25,35 +25,32 @@ namespace lczero {
 
 class Transforms {
  public:
-  
-  
   static constexpr auto kWinogradAlpha = 4;
   static constexpr auto kWinogradTile = kWinogradAlpha * kWinogradAlpha;
```

Comment:
perhaps these can be moved to blas.h? (not a problem with current PR)

Reply:
Yes, they are both an acronyms (Basic Linear Algebra Sub-routine) without using google :)
 I would prefer  OpenCL than CL for the folder inside neural.